src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -183,7 +183,7 @@ SpinelNCPControlInterface::config_gateway(bool defaultRoute, const uint8_t prefi
     const static int kDefaultRouteFlag = 1 << 1;
     const static int kDhcpFlag = 1 << 3; (void)kDhcpFlag;
 
-	spinel_ipv6addr_t addr = {};
+	struct in6_addr addr = {};
 	uint8_t flags = 0;
```

Comment:
>If I am not mistaken `spinel_ipv6adr_t` is typedefed as `struct in6_addr` in spinel.h

Not anymore. Had to typedef it to a `struct { uint8_t bytes[16]; }` because `struct in6_addr` isn't available on all OpenThread platforms. See https://github.com/openthread/openthread/issues/121